core/src/test/scala/unit/kafka/server/OffsetFetchRequestTest.scala

```diff
@@ -126,6 +126,56 @@ class OffsetFetchRequestTest extends BaseRequestTest {
     }
   }
 
+  @Test
+  def testOffsetFetchRequestAllOffsetsSingleGroup(): Unit = {
```

Comment:
This one is based on `testOffsetFetchRequestSingleGroup`. I kept the exact same assertions.